horace_core/sqw/coord_transform/@ortho_proj/ortho_proj.m

```diff
@@ -11,13 +11,21 @@
     %   >> proj = ortho_proj(proj_struct)
     %             where proj_struct is the
     %             structure, containing any fields, with names, equal any
-    %             public fields of the ortho_proj class
+    %             public fields of the ortho_proj class.
+    %
+    % As a standard serializable class accepts full set of positional and
+    % key-value parameters, which constitute its properties
```

Comment:
I'm not clear why you need to see this. Surely this is true for all serializable classes. What particular information are you pointing to with this generalisation.

Reply:
I am reminding the obvious and in the next sentence proivde the exact form of the constructor